docs/content/patterns/crud-data-tables/index.mdx

```diff
@@ -0,0 +1,448 @@
+								</TextLink>
+								<TextLink aria-describedby="ABC123456" href="#">
+									Delete
+								</TextLink>
+							</Flex>
+						</TableCell>
+					</TableRow>
+				</TableBody>
+			</Table>
+		</TableWrapper>
+		<ButtonGroup>
+			<Button variant="secondary" size="sm" iconBefore={PlusIcon}>
+				Add record
+			</Button>
+		</ButtonGroup>
+	</Stack>
+</Stack>
+```
+
+### Read a record in a data table
+
+Provide a link in the first column of the table so users can open and read a record. The record number or other unique identifier should be made bold and used for the link label.
+
+Provide a side drawer or side page to provide the record information. Use side drawers for small records or side pages for longer records that a user will need to scroll.
+
+After a user has read the record return them to the table page.
+
+Reading a record in a table is repeatable and scalable pattern.
+
+### Update a record in a data table
+
+Provide an ‘Edit’ link in the last column of the table so users can open and update or edit existing records. The edit link can be included in a drop down of actions in the last column if there are more than two actions required.
+
+Editing content in place in semantic tables is difficult for some users, so use a side page or side drawer to provide all inputs and controls required to edit the record. When a user elects to edit a record open the side drawer or side page and show the populated fields to be edited.
+
+After the record is edited in the side page or drawer, return the user to the table page, show the edited record in the table, and provide a Section alert immediately above the table to show success. Focus the Section alert so the change is announced to screen reader users.
+
+The update or edit pattern is repeatable and scalable.
+
+<ImageWithBorder
+	alt="Screenshot of an open side drawer with the title ‘Add record’ and a populated short form with text input, radio buttons, a Select component and form buttons ‘Add record’ and ‘Cancel’"
+	src="/img/patterns/crud-tables-edit-record.png"
+/>
+
+### Delete a record from a data table
+
+Provide a ’Delete’ link in the last column of the table so users can delete existing records. The delete link can be included in a drop down of actions in the last column if there are more than two actions required.
+
+If the table supports batch actions more than one record can be deleted at the same time.
+
+Before any record is deleted provide a destructive action modal to make sure a user intends to remove the record.
+
+After the record is deleted, remove the record from the table, and provide a Section alert immediately above the table to show success. Focus the Section alert so the change is announced to screen reader users.
+
+The delete pattern is repeatable and scalable.
+
+<ImageWithBorder
+	alt="Screenshot of an open Modal dialog with the title ‘Are you sure you want to remove this record’ and description ‘It will be removed immediately. You can't undo this action.’"
```

Comment:
I think the image should be updated accordingly
```suggestion
	alt="Screenshot of an open Modal dialog with the title ‘Are you sure you want to delete this record?’ and description ‘It will be deleted immediately. You can't undo this action.’"
```